ocs_ci/ocs/resources/pod.py

```diff
@@ -1660,12 +1660,18 @@ def get_osd_removal_pod_name(osd_id, timeout=60):
         str: The osd removal pod name
 
     """
+    ocp_version = float(get_ocp_version())
+    if ocp_version >= 4.7:
+        pattern = "ocs-osd-removal-job"
+    else:
+        pattern = f"ocs-osd-removal-{osd_id}"
```

Comment:
We need a verification on ocs 4.6 build as will to make sure everything is working fine